tests/unit/test_decrypt_with_shares.py

```diff
@@ -261,7 +261,7 @@ def test_decrypt_ballot_with_missing_guardians(self):
 
         compensated_shares = {
             available_guardian.id: compute_compensated_decryption_share_for_ballot(
-                available_guardian.share_key(),
+                available_guardian._election_keys,
```

Comment:
May want to move this line and the similar lines to the `Arrange` since the election_keys are not public and the guardian class would normally be doing it internally. Realistically, this could be done just by manually creating key_pairs instead.